general/clients/index.md

```diff
@@ -107,7 +107,8 @@ The official Jellyfin Roku app.
 
 **Links:**
 
-- [Roku Channel Store](https://my.roku.com/add/jellyfin).
+<a href='https://channelstore.roku.com/details/592369/jellyfin'><img alt='Get it on the Roku Store' src='https://www.roku.com/s/1597964107537/fonts/roku-logo.svg' width="200"/></a>
```

Comment:
Looks like the image isn't working anymore. I think the number part of this URL changes regularly.

Reply:
It still works for me?

Reply:
Oh, yeah. There is one on Wikipedia that I can link (as it claims it's under public domain). If that's ok to do.

Reply:
Just put it in the repository please